build.sbt

```diff
@@ -129,11 +130,20 @@ lazy val openlawCore = crossProject(JSPlatform, JVMPlatform)
   .settings(releaseSettings: _*)
   .enablePlugins(WartRemover)
 
+
+lazy val version = "sbt version"!
+
+addCommandAlias("releaseCore", ";project openlawCore ;release release-version ${version} next-version ${version-SNAPSHOT} with-defaults")
+addCommandAlias("releaseCoreJS", ";project openlawCoreJS ;release release-version ${version} next-version ${version-SNAPSHOT} with-defaults")
```

Comment:
Mostly curious, but can these recurse? Like, could we then add a third alias that did `addCommandAlias("releaseBoth", "releaseCore releaseCoreJs")`? (Maybe not worth it unless it specifically let us override `'release'` more just curious for future usage.

Reply:
yes I think so. I like that idea, i'll try it out